templates/example-webgpu.html

```diff
@@ -8,7 +8,7 @@
       <h2 class="example__title">{{ category.title }} / {{ page.title }}</h2>
       <a class="example__back" href="/examples-webgpu"><i class="icon icon--chevron-left"></i> Back to examples</a>
       <a class="example__github"
-          href="https://github.com/bevyengine/bevy/blob/latest/{{ page.extra.github_code_path }}">
```

Comment:
This was intentional. Editor config requires the tab amount to be a multiple of 2 to keep tab consistency.

Reply:
My local copy of djlint removed the space. Will add it back in manually.